core/objects.go

```diff
@@ -587,6 +582,9 @@ type rawSignedCertificateTimestamp struct {
 	Extensions string `json:"extensions"`
 }
 
+// UnmarshalJSON parses a simplified JSON form of an SCT, covering all of
```

Comment:
Not quite accurate that this is a simplified JSON form, how about this comment:

UnmarshalJSON parses the add-chain response from a CT log. It fills all of the fields in the SignedCertificateTimestamp struct except for ID and CertificateSerial, which are used for local recordkeeping in the Boulder DB.